tests/test_community.py

```diff
@@ -202,11 +202,11 @@ def test_community_join_post(client):
             follow_redirects=True,
         )
         assert (
-            b'{"access_token":"RefreshToken","message":"Community Joined","refresh_token":"AccessToken","status_code":200}\n'
+            b'{"access_token":"RefreshToken","data":{"Members":[],"name":"mock"},"message":"Community Joined","refresh_token":"AccessToken","status_code":200}\n'
```

Comment:
is this test case correct? the community returns with no members after a `join` request?

Reply:
Get is already mocked to have an empty members list, so when get is called to return again it pulls the empty member list

Reply:
okay, that makes sense but it kinda makes this test semi redundant right?

Reply:
A bit, this is just to mainly verify the route works. The DB tests verify the update functionality

Reply:
We can fix that, but I think for the time that works